packages/modules/providers/auth-emailpass/src/services/emailpass.ts

```diff
@@ -35,14 +39,97 @@ export class EmailPassAuthService extends AbstractAuthModuleProvider {
     this.logger_ = logger
   }
 
-  protected async createAuthIdentity({ email, password, authIdentityService }) {
+  protected async hashPassword(password: string) {
     const hashConfig = this.config_.hashConfig ?? { logN: 15, r: 8, p: 1 }
     const passwordHash = await Scrypt.kdf(password, hashConfig)
+    return passwordHash.toString("base64")
+  }
+
+  async generateResetPasswordToken(
```

Comment:
q: should this be a generic temporary auth token generator? That way we can also reuse this for things like "magic link login"

Reply:
Yup, currently working on this – @sradevski also pointed this out in Slack.

Reply:
nice!